src/java/org/apache/cassandra/index/sai/disk/v1/V1OnDiskFormat.java

```diff
@@ -235,7 +235,7 @@ public int openFilesPerSSTableIndex()
     @Override
     public int openFilesPerColumnIndex(IndexContext indexContext)
     {
-        // For the V1 format there are always 2 open files per index - index (kdtree or terms) + postings
+        // For the V1 format there are always 2 open files per index - index (balanced tree or terms) + postings
```

Comment:
For the `bbtree`, it's just the auxiliary postings in the second file, right?

Reply:
Correct. I will add a note to this effect.

Reply:
Looking at `BlockBalancedTreePostingsWriter`, it is both leaf and aux postings in that file. :D

Reply:
I think we only need to mention that one of the files here is the postings file. This method is about how many files are open for an index. It doesn't need to go into specifics of what are in the files. We do that elsewhere.